device/mellanox/x86_64-nvidia_sn5600-r0/Mellanox-SN5600-O128/port_config.ini

```diff
@@ -15,133 +16,133 @@
+Ethernet368    368,369,370,371                        etp47a    47       400000
+Ethernet372    372,373,374,375                        etp47b    47       400000
+Ethernet376    376,377,378,379                        etp48a    48       400000
+Ethernet380    380,381,382,383                        etp48b    48       400000
+Ethernet384    384,385,386,387                        etp49a    49       400000
+Ethernet388    388,389,390,391                        etp49b    49       400000
+Ethernet392    392,393,394,395                        etp50a    50       400000
+Ethernet396    396,397,398,399                        etp50b    50       400000
+Ethernet400    400,401,402,403                        etp51a    51       400000
+Ethernet404    404,405,406,407                        etp51b    51       400000
+Ethernet408    408,409,410,411                        etp52a    52       400000
+Ethernet412    412,413,414,415                        etp52b    52       400000
+Ethernet416    416,417,418,419                        etp53a    53       400000
+Ethernet420    420,421,422,423                        etp53b    53       400000
+Ethernet424    424,425,426,427                        etp54a    54       400000
+Ethernet428    428,429,430,431                        etp54b    54       400000
+Ethernet432    432,433,434,435                        etp55a    55       400000
+Ethernet436    436,437,438,439                        etp55b    55       400000
+Ethernet440    440,441,442,443                        etp56a    56       400000
+Ethernet444    444,445,446,447                        etp56b    56       400000
+Ethernet448    448,449,450,451                        etp57a    57       400000
+Ethernet452    452,453,454,455                        etp57b    57       400000
+Ethernet456    456,457,458,459                        etp58a    58       400000
+Ethernet460    460,461,462,463                        etp58b    58       400000
+Ethernet464    464,465,466,467                        etp59a    59       400000
+Ethernet468    468,469,470,471                        etp59b    59       400000
+Ethernet472    472,473,474,475                        etp60a    60       400000
+Ethernet476    476,477,478,479                        etp60b    60       400000
+Ethernet480    480,481,482,483                        etp61a    61       400000
+Ethernet484    484,485,486,487                        etp61b    61       400000
+Ethernet488    488,489,490,491                        etp62a    62       400000
+Ethernet492    492,493,494,495                        etp62b    62       400000
+Ethernet496    496,497,498,499                        etp63a    63       400000
+Ethernet500    500,501,502,503                        etp63b    63       400000
+Ethernet504    504,505,506,507                        etp64a    64       400000
+Ethernet508    508,509,510,511                        etp64b    64       400000
 Ethernet512    512                                    etp65     65       25000
```

Comment:
Please check. There was no such port on the file before the chnage

Reply:
Ethernet512 is not part of the diff and exists even without my changes